internal/controllers/addon/olm_reconciler.go

```diff
@@ -86,6 +86,7 @@ func (r *olmReconciler) Reconcile(ctx context.Context,
 	} else if requeueResult != resultNil {
 		return handleExit(requeueResult), nil
 	}
+	reportLastObservedAvailableCSV(addon, currentCSVKey.Name)
```

Comment:
```suggestion
	reportLastObservedAvailableCSV(addon, currentCSVKey)
```
shall we store `namespace/name` instead of just `name`. I think i will provide more context. In addition, it is a more actionable piece of information. 🧑‍💻

Reply:
would this function call fit better inside the `observeCurrentCSV()` block